### DISCUSSION TITLE
Daggerboard installation error

### DISCUSSION THREAD

**Discussion by trailblazergt** [nyph-infosec/daggerboard#27] [General]
Hi all,
While installing the latest Daggerboard binary version on Ubuntu 24.04 LTS I am getting this error:
`>>> Creating superuser admin ...

Traceback (most recent call last):
  File "/var/www/Daggerboard/manage.py", line 15, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/var/www/Daggerboard/manage.py", line 26, in <module>
    main()
  File "/var/www/Daggerboard/manage.py", line 17, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?

=================================================
Daggerboard installed, please reboot your machine
=================================================

Can you please help me out with this?

**Comment by kojak-a** [comment]
Hi @trailblazergt, please refer to the readme for installation instructions. The operating system dependency requires Ubuntu 18 or 20.04.
